### ISSUE TITLE
Issue on "Iron out the creases"

### ISSUE THREAD

**Issue by gkavboy** [liamcain/obsidian-creases#25]
I have some note, like this
```md
# Сhapter 1 %% fold %%
Text of chapter 1.
Another text.
## Section 1.1
Text of section 1.1
Another text.
## Section 1.2
Text of section 1.2
Another text.
```
![изображение](https://user-images.githubusercontent.com/25064847/163185747-e3986ba5-f501-41bf-bc41-5d1aad3bb0a0.png)

When I run the command "Iron out the creases" it break my note like this:
```
# Сhapter 1Text of chapter 1.
Another text.
## Section 1.1
Text of section 1.1
Another text.
## Section 1.2
Text of section 1.2
Another text.
```

![изображение](https://user-images.githubusercontent.com/25064847/163189119-3b90b996-4cb3-441b-beb3-1215312ff088.png)

The header first line subtext join to the header.
It look like deteting the line break char on ending of header.

**Comment by gkavboy** [comment]
If "Chapter 1" do not have a subtext and have only nested headers all works good

P.S. The plugin is cool, thanks!

**Comment by liamcain** [comment]
Fixed in v0.6.2
